CNV-50110: Add a small migration section and link to MTV docs (#85634)
openshift/openshift-docs: modules/migrating-to-virt.adoc

// Module included in the following assemblies:
//
// * virt/getting_started/virt-getting-started.adoc

:_mod-docs-content-type: PROCEDURE

> 

Reviewer: This may not be an issue at all, but there usually isn't a blank line here.

[id="migrating-to-virt_{context}"]
= Migrating to {VirtProductName}

> To migrate virtual machines from an external provider such as VMware vSphere, Red Hat OpenStack, Red Hat Virtualization, or another {VirtProductName} cluster, use the Migration Toolkit for Virtualization (MTV). You can also migrate Open Virtual Appliance (OVA) files created by VMware vSphere.

Reviewer: There are some attributes in the `_attributes/common-attributes.adoc` file that you can use here (and elsewhere in the PR where these phrases come up). I would also suggest adding a set of attributes for MTV to the common-attributes file. You can model it after the MTC attributes:

```
:mtc-first: Migration Toolkit for Containers (MTC)
:mtc-short: MTC
:mtc-full: Migration Toolkit for Containers
```

Note on the following suggestion: we are advised not to use the "commit suggestion" button in the UI, but rather to make any changes manually. The "suggestion" interface is still the best way to propose changes to existing text, which is why we use it anyway. :) 
```suggestion
To migrate virtual machines from an external provider such as {vmw-first}, {rh-openstack-first}, Red Hat Virtualization, or another {product-title} cluster, use the Migration Toolkit for Virtualization (MTV). You can also migrate Open Virtual Appliance (OVA) files created with {vmw-full}.
```

Regarding `{VirtProductName} cluster`: more accurate phrasing would be `{product-title} cluster` (rendering as "OpenShift Container Platform cluster" in the enterprise distro).

[NOTE]
====

> Migration Toolkit for Virtualization is not part of {VirtProductName} and requires separate installation. Because of that, all links in this procedure lead outside of {VirtProductName} documentation.

Reviewer: Minor phrasing suggestion and a nit:
```suggestion
Migration Toolkit for Virtualization is not part of {VirtProductName} and requires separate installation. For this reason, all links in this procedure lead outside of the {VirtProductName} documentation.
```

====

.Prerequisites

> * The Migration Toolkit for Virtualization Operator link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.7/html/installing_and_using_the_migration_toolkit_for_virtualization/installing-the-operator_mtv[is installed].

Reviewer: I'm inclined to use an attribute for the MTV version as well; otherwise, we would need to manually update the version number embedded in these links for each release. You could alternatively use "latest" instead of 2.7, but please make sure that's supportable first.

The common-attributes file has these for MTC:

```
:mtc-version: 1.8
:mtc-version-z: 1.8.5
```

.Procedure
. link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.7/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-vms-web-console_mtv#adding-source-providers[Configure the source provider].
+

> Select VMware vSphere, Red Hat OpenStack, Red Hat Virtualization, OVA files created by VMware vSphere, or another {VirtProductName} cluster as the source provider. Depending on the selection, enter required information such as the URL of the API endpoint or credentials needed to access the virtual machines.

Reviewer: This might be my personal opinion, but I don't think we need to mention the list of source providers here. Suggestion for minimalism:

```suggestion
Select the source provider and enter any required information, such as the URL of the API endpoint or credentials needed to access the virtual machines.
```

. link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.7/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-vms-web-console_mtv#creating-migration-plans-ui[Create a migration plan].
+
Select from the list of available virtual machines, configure the target provider, and define storage and network mappings.

. link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.7/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-vms-web-console_mtv#running-migration-plan_mtv[Run the migration plan].
+
Watch the progress, view migration logs, or cancel the migration.

. Optional: link:https://docs.redhat.com/en/documentation/migration_toolkit_for_virtualization/2.7/html/installing_and_using_the_migration_toolkit_for_virtualization/migrating-vms-web-console_mtv#migration-plan-options-ui_mtv[Archive the migration plan].
+

> After confirming that the migration has completed successfully, you can archive the migration plan to delete the logs and clean up all temporary resources created during migration. Once archived, you can permanently delete the migration plan.

Reviewer: The IBM Style Guide dissuades using "once" to mean "after." 
> Avoid terms that might be misinterpreted. For example, use _once_ to mean _one time_, not to mean _after_ or _when_. Use _since_ in relation to the passing of a period of time, not as a synonym for _because_.

Reference: https://www.ibm.com/docs/en/ibm-style?topic=medium-global-audiences

```suggestion
After confirming that the migration has completed successfully, you can archive the migration plan to delete the logs and clean up all temporary resources created during migration. You can permanently delete an archived migration plan.
```
